PlaceRouteHierFlow/placer/ILP_solver.cpp

```diff
@@ -1042,8 +1042,8 @@ bool ILP_solver::FrameSolveILP(design& mydesign, SeqPair& curr_sp, PnRDB::Drc_in
   y_pitch = drcInfo.Metal_info[h_metal_index].grid_unit_y;
 
   // each block has 4 vars, x, y, H_flip, V_flip;
-  unsigned int N_var = mydesign.Blocks.size() * 4 + mydesign.Nets.size() * 2;
-  // i*4+1: x
+  unsigned int N_var = (mydesign.Blocks.size() + mydesign.Nets.size()) * 4;
+  // i*4+1:x
```

Comment:
Why times 4 here?

Reply:
one variable for each size of the "bounding box" of net. LL_x, LL_y, UR_x and UR_y.

Reply:
good. Thanks.